client/src/modules/fiscal/templates/credit.cell.tmpl.html

```diff
@@ -0,0 +1 @@
+<div class="ui-grid-cell-contents text-right">{{row.entity.credit}}</div>
```

Comment:
There is no need to make a separate template for this - it will only slow down rendering.  Just add `cellClass : 'text-right'` to your columDef in the controller.